topdown/http.go

```diff
@@ -104,6 +105,12 @@ const (
 
 	// HTTPSendNetworkErr represents a network error.
 	HTTPSendNetworkErr string = "eval_http_send_network_error"
+
+	// minRetryDelay is amount of time to backoff after the first failure.
+	minRetryDelay = time.Millisecond * 100
+
+	// maxRetryDelay is the upper bound of backoff delay.
+	maxRetryDelay = time.Second * 120
```

Comment:
Isn't this a very long max delay? Is it ever ok for a policy to take minutes to execute?

Reply:
I wasn't sure what a good upper bound would be. I'll update it to a minute but I think eventually we want to make this configurable. Also the user has control on the number of retry attempts so they control the accepted latency here. LMKWYT?

Reply:
Yes, I agree that as long as the user can have some influence over the delay, it's not of great concern 👍 . 
Perhaps we should have an indication in the docs as to what measure the configured retry count will affect the delay. I haven't looked at how the actual delay is determined based on the attempt count, so I don't know how hard it would be to put that into words, though.

Reply:
I've added a note in the docs that the delay is based on an exponential backoff mechanism.